test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -290,7 +293,25 @@ public void TestBuiltInFunctionRecognition()
             Assert.IsFalse(libraryServices.IsFunctionBuiltIn(categoryName, nickName));
 
         }
+        
+        [Test]
+        [Category("UnitTests")]
+        public void GetAllFunctionDescriptorsTest()
```

Comment:
@yeexinc the nodes can change over time so this test may not hold true when a new overload is added for any of these nodes for example. We usually test the library services against a dummy library in the test framework called `FFITarget.dll`. See other tests using this library.